src/explore-education-statistics-admin/src/pages/admin-dashboard/components/FeedbackDetailsModal.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import ButtonText from '@common/components/ButtonText';
+import Modal from '@common/components/Modal';
+import { FeedbackViewModel } from '@common/services/types/feedback';
+import FormattedDate from '@common/components/FormattedDate';
+
+interface Props {
+  feedback: FeedbackViewModel;
+  getResponseText: (
+    response: FeedbackViewModel['response'],
+  ) => 'Useful' | 'Not useful' | 'Problem encountered' | '';
```

Comment:
Don't need to specify the exact string literals as we're not doing any logic that requires this level of type safety. 

A type of `string` is sufficient and will make it easier to change the labels in the future.